tests/spread/core24/cross-compile/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Cross-compile a snap with build-packages and stage-packages from a foreign architecture.
+
+restore: |
+  snapcraft clean
+  rm -f ./*.snap
+
+execute: |
+  snapcraft pack
```

Comment:
worth verifying the packed output to see if it is of the correct architecture

Reply:
Good idea, I added a check.